client/src/panels/panel_declarations/results/results_intro.js

```diff
@@ -0,0 +1,143 @@
+        <TM k="dp_summary_text" args={summary_text_args} />
+        <TM k="drr_summary_text" args={summary_text_args} />
+        {summary_result_counts.drr_results > 0 && <TM k="reports_links_text" args={doc_urls} />}
+      </div>
+    </div>
+  );
+};
+
+export const declare_results_intro_panel = () => declare_panel({
+  panel_key: "results_intro",
+  levels: ["gov", "dept"],
+  panel_config_func: (level, panel_key) => ({
+    requires_granular_result_counts: true,
+    footnotes: ["RESULTS_COUNTS", "RESULTS"],
+    source: (subject) => get_source_links(["DP","DRR"]),
+    calculate: (subject) => {
+
+      const is_gov = subject.level == 'gov';
+
+      const verbose_counts = (() => {
+        if(is_gov){
+          const dept_counts = ResultCounts.get_all_dept_counts();
+          const counts_by_dept = _.chain(dept_counts)
+            .map( row => ({
+              subject: Dept.lookup(row.id),
+              counts: row,
+            }))
+            .map( obj => ({...obj, total: d3.sum(_.values(obj.counts)) } ) )
+            .value();
+          const gov_counts = _.mergeWith({}, ...dept_counts, (val, src) => _.isNumber(val) ? val + src : src);
+          const depts_with_dps = _.sumBy(counts_by_dept, dept => dept.counts[`${latest_dp_doc_key}_results`] > 0 ? 1 : 0);
+          const depts_with_drrs = _.sumBy(counts_by_dept, dept => dept.counts[`${latest_drr_doc_key}_results`] > 0 ? 1 : 0);
+          return {
+            depts_with_dps,
+            depts_with_drrs,
+            ...(_.omit(gov_counts, ['id','level','subject_id'])),
+          };
+        } else {
+          return {
+            num_crs: _.size(subject.crsos),
+            num_programs: _.reduce(subject.crsos, (sum,crso) => sum+_.size(crso.programs), 0),
+            ...ResultCounts.get_dept_counts(subject.id),
+          };
+        }
+      })();
+
+      if(verbose_counts[`${latest_dp_doc_key}_results`] < 1){
+        return false;
+      }
+
+      const summary_result_counts = {
+        dp_results: verbose_counts[`${latest_dp_doc_key}_results`],
+        dp_indicators: verbose_counts[`${latest_dp_doc_key}_indicators`],
+        drr_results: verbose_counts[`${latest_drr_doc_key}_results`],
+        drr_indicators: verbose_counts[`${latest_drr_doc_key}_total`],
+        num_crs: is_gov ? 0 : verbose_counts.num_crs,
+        num_programs: is_gov ? 0 : verbose_counts.num_programs,
+        depts_with_dps: is_gov ? verbose_counts.depts_with_dps : 1,
+        depts_with_drrs: is_gov ? verbose_counts.depts_with_drrs : 1,
+      };
+
+      const doc_urls = {
+        dp_url: result_docs[latest_dp_doc_key][`doc_url_${window.lang}`],
+        drr_url: result_docs[latest_drr_doc_key][`doc_url_${window.lang}`],
+      };
+
+      return {
+        subject,
+        is_gov,
+        summary_result_counts,
+        doc_urls,
+      };
+    },
+    render({ calculations, sources}){
+      const {
+        subject,
+        panel_args,
```

Comment:
This `subject` should be the same as the `subject` on `panel_args`, right? This and line `136` may be redundant. Minor, but gave me pause and I could see future developers also stopping to wonder wether or not these are supposed to be the same subjects too.